README.md

```diff
@@ -1,53 +1,30 @@
-![spack-stack-logo](https://user-images.githubusercontent.com/8006981/234488735-45b2c5fa-1de6-47ad-ae3b-4a6829ae49b9.png)
-
-Spack-stack enables the installation of software required
-for HPC system deployments of NOAA's Unified Forecast System (UFS) and
-other weather and climate models, including components of the Joint
-Effort for Data assimilation Integration (JEDI).
-
-Spack-stack is a collaborative effort between:
-* [NOAA Environmental Modeling Center (EMC)](https://www.emc.ncep.noaa.gov/emc_new.php)
-* [UCAR Joint Center for Satellite Data Assimilation (JCSDA)](https://www.jcsda.org/)
-* [Earth Prediction Innovation Center (EPIC)](https://epic.noaa.gov/).
-
-Spack-stack is a thin layer around a fork of the
-[spack](https://github.com/spack/spack) repository. Spack is a
-community-supported, multi-platform, Python-based package manager
-originally developed by the Lawrence Livermore National Laboratory
+<img src="https://user-images.githubusercontent.com/8006981/234488735-45b2c5fa-1de6-47ad-ae3b-4a6829ae49b9.png" width="425">
+
+Spack-stack is a framework for installing software libraries 
+to support NOAA's Unified Forecast System (UFS) and the Joint
+Effort for Data assimilation Integration (JEDI) applications on a range of
+R&D and operational platforms.
```

Comment:
We need to add a bit to this paragraph. spack-stack not only builds requirements for JEDI and UFS applications, but also for other models that get coupled with JEDI (MPAS, NEPTUNE, very soon GEOS). How about
```
Spack-stack is a framework for installing software libraries to support NOAA's Unified Forecast System (UFS) ecosystem and the Joint Effort for Data assimilation Integration (JEDI) coupled to several Earth system prediction models (MPAS, NEPTUNE, UM, GEOS).

Spack-stack is designed to work seamlessly on a variety of R&D and operational platforms. It provides ...
```

Reply:
Okay, see what you think